app/src/main/java/science/apolline/view/activity/SplashScreen.kt

```diff
@@ -322,8 +286,8 @@ class SplashScreen : RootActivity(), AnkoLogger {
                 nameOrcode = device.address.toString()
             } else {
                 nameOrcode = device.name.toString()
-                if (nameOrcode.toLowerCase().contains(regex = "^ioio.".toRegex()) ||
-                        nameOrcode.toLowerCase().contains(regex = "^appa.".toRegex())) {
+                if (nameOrcode.toLowerCase().contains(regex = R.string.ioio_regex.toString().toRegex()) ||
```

Comment:
There is no point in doing this:
* Text for end users should be localised
* Internal encodings should be documented as constants